src/clientiface.cpp

```diff
@@ -252,7 +252,8 @@ void RemoteClient::GetNextBlocks (
 			*/
 
 			float camera_fov = (72.0*M_PI/180) * 4./3.;
-			if(isBlockInSight(p, camera_pos, camera_dir, camera_fov, 10000*BS) == false)
+			if(isBlockInSight(p, camera_pos, camera_dir, camera_fov,
+					full_d_max * BS * (MAP_BLOCKSIZE + 1)) == false)
```

Comment:
``full_d_max * BS * MAP_BLOCKSIZE` or whatever it needs to be can be calculated at line 177 to avoid calculatingit multiple times?